src/nvim/version.c

```diff
@@ -56,2432 +56,4205 @@ char *version_cflags = "Compilation: " NVIM_VERSION_CFLAGS;
+    // 3614,
+    3613, 3612, 3611, 3610, 3609,
+    // 3608,
+    3607,
+    // 3606, 3605,
+    3604, 3603, 3602, 3601, 3600, 3599, 3598,
+    // 3597, 3596,
+    3595,
+    // 3594,
+    3593,
+    // 3592,
+    3591,
+    // 3590, 3589,
+    3588, 3587, 3586,
+    // 3585,
+    3584, 3583, 3582, 3581, 3580, 3579,
+    // 3578,
+    3577, 3576, 3575, 3574, 3573, 3572, 3571, 3570,
+    // 3569,
+    3568, 3567, 3566, 3565, 3564, 3563,
+    // 3562,
+    3561,
+    // 3560, 3559, 3558, 3557,
+    3556, 3555,
+    // 3554, 3553, 3552,
+    3551, 3550, 3549,
+    // 3548,
+    3547, 3546, 3545,
+    // 3544,
+    3543, 3542, 3541, 3540, 3539,
+    // 3538,
+    3537,
+    // 3536, 3535,
+    3534,
+    // 3533,
+    3532, 3531, 3530,
+    // 3529,
+    3528, 3527, 3526, 3525,
+    // 3524, 3523,
+    3522, 3521, 3520, 3519, 3518, 3517,
+    // 3516,
+    3515, 3514, 3513, 3512,
+    // 3511,
+    3510, 3509,
+    // 3508,
+    3507,
+    // 3506,
+    3505,
+    // 3504, 3503, 3502,
+    3501, 3500, 3499, 3498, 3497,
+    // 3496,
+    3495, 3494, 3493, 3492, 3491, 3490, 3489, 3488, 3487, 3486,
+    // 3485,
+    3484, 3483, 3482,
+    // 3481,
+    3480,
+    // 3479,
+    3478, 3477, 3476, 3475,
+    // 3474,
+    3473, 3472, 3471, 3470, 3469, 3468, 3467, 3466, 3465, 3464, 3463, 3462, 3461, 3460, 3459, 3458,
+    // 3457, 3456,
+    3455, 3454, 3453, 3452, 3451, 3450, 3449, 3448,
+    // 3447,
+    3446,
+    // 3445, 3444, 3443, 3442, 3441,
+    3440, 3439, 3438, 3437,
+    // 3436, 3435,
+    3434, 3433, 3432,
+    // 3431,
+    3430,
+    // 3429,
+    3428,
+    // 3427,
+    3426, 3425, 3424,
+    // 3423, 3422,
+    3421,
+    // 3420,
+    3419,
+    // 3418,
+    3417, 3416, 3415, 3414,
+    // 3413, 3412, 3411,
+    3410, 3409, 3408, 3407,
+    // 3406, 3405, 3404,
+    3403, 3402,
+    // 3401,
+    3400, 3399, 3398, 3397,
+    // 3396,
+    3395, 3394, 3393, 3392, 3391, 3390, 3389, 3388, 3387, 3386, 3385, 3384,
+    // 3383,
+    3382, 3381,
+    // 3380,
+    3379, 3378,
+    // 3377, 3376,
+    3375, 3374,
+    // 3373,
+    3372,
+    // 3371, 3370,
+    3369, 3368,
+    // 3367, 3366, 3365, 3364,
+    3363, 3362,
+    // 3361,
+    3360,
+    // 3359,
+    3358, 3357,
+    // 3356,
+    3355, 3354,
+    // 3353, 3352, 3351, 3350, 3349,
+    3348,
+    // 3347, 3346,
+    3345,
+    // 3344, 3343, 3342, 3341, 3340, 3339,
+    3338, 3337, 3336,
+    // 3335,
+    3334, 3333, 3332, 3331, 3330, 3329, 3328, 3327,
+    // 3326,
+    3325,
+    // 3324, 3323, 3322,
+    3321,
+    // 3320,
+    3319,
+    // 3318, 3317, 3316, 3315,
+    3314, 3313,
+    // 3312,
+    3311, 3310,
+    // 3309, 3308, 3307,
+    3306,
+    // 3305, 3304, 3303,
+    3302,
+    // 3301, 3300, 3299,
+    3298,
+    // 3297, 3296,
+    3295, 3294, 3293, 3292, 3291,
+    // 3290,
+    3289,
+    // 3288, 3287,
+    3286, 3285, 3284, 3283, 3282,
+    // 3281,
+    3280,
+    // 3279, 3278, 3277, 3276, 3275, 3274, 3273, 3272, 3271, 3270, 3269, 3268, 3267, 3266,
```

Comment:
Was expecting these lines to be modified after `make formatc`.
Were comments excluded for hard wrap?